### ISSUE TITLE
Multi detection

### ISSUE THREAD

**Issue by Averu-zz** [bgshih/aster#69]
Is it possible to detect and recognise multiple texts from a scene?

**Comment by dsandii** [comment]
If you mean predict, for instance, 1 text box per word in the image then yes, it is possible.

**Comment by Averu-zz** [comment]
Yes. Predict 1 text box for each word that appears in the image and then predict the contents of each textbox. Where in the files should i look to be able to do that?

**Comment by dsandii** [comment]
Nevermind, I might have been wrong, apologies.

**Comment by dsandii** [comment]
I found something that may hopefully help you.
In `aster/protos/train.proto` the 8th defined field around line 35 called `from_detection_checkpoint` is an optional bool that allows you to specify if your `fine_tune_checkpoint` was from an object detection model.

This appears to default to `False`, meaning the default training configuration assumes that you're using a CNN instead of an R-CNN.

Hopefully you're able to configure this and make it work. If you find a pre-trained detection model that ends up working, please let us know.
